packages/voorbeeld-design-tokens/figma/voorbeeld.tokens.json

```diff
@@ -2821,6 +2821,68 @@
           "$value": "{voorbeeld.box-shadow.lg}"
         }
       }
+    },
+    "utrecht": {
+      "drawer": {
+        "border-width": {
+          "$type": "borderWidth",
+          "$value": "{voorbeeld.border-width.sm}"
+        },
+        "padding-inline-start": {
+          "$type": "spacing",
+          "$value": "{voorbeeld.space.inline.mouse}"
+        },
+        "padding-inline-end": {
+          "$type": "spacing",
+          "$value": "{voorbeeld.space.inline.mouse}"
+        },
+        "background-color": {
+          "$type": "color",
+          "$value": "{utrecht.document.background-color}"
+        },
+        "border-color": {
+          "$type": "color",
+          "$value": "{voorbeeld.container.border-color}"
+        },
+        "color": {
+          "$type": "color",
+          "$value": "{utrecht.document.color}"
+        },
+        "border-radius": {
+          "$type": "borderRadius",
+          "$value": "0px"
+        },
+        "max-block-size": {
+          "$type": "sizing",
+          "$value": "800px"
+        },
+        "max-inline-size": {
+          "$type": "sizing",
+          "$value": "1440px"
+        },
+        "min-inline-size": {
+          "$type": "sizing",
+          "$value": "312px"
+        },
+        "min-block-size": {
+          "$type": "sizing",
+          "$value": "0px"
```

Comment:
min-block-size: 0 lijkt me nogal klein

het idee van min-block-size is dat die alleen van toepassing is op de horizontale variant